Gemfile.lock

```diff
@@ -35,7 +35,7 @@ GEM
       rack-test (>= 0.5.4)
       xpath (~> 2.0)
     cert (0.1.0)
-      fastlane_core (~> 0.2.0)
+      fastlane_core
```

Comment:
What happened here? Maybe a missing `bundle install`? Anyway, shouldn't matter, since it's overwritten on the next release anyway afaik.